src/devices/develco.ts

```diff
@@ -728,8 +728,9 @@ const definitions: Definition[] = [
         exposes: [e.occupancy(), e.battery_low()],
     },
     {
-        zigbeeModel: ['HMSZB-110'],
-        model: 'HMSZB-110',
+        whiteLabel: [{vendor: 'Frient', model: 'HMSZB-120'}, {vendor: 'Frient', model: 'HMSZB-110'}],
```

Comment:
Please keep `model` as is and only add `HMSZB-120` as a whitelabel with a fingerprint containing a `zigbeeModel`